tasks/main.yml

```diff
@@ -21,7 +21,7 @@
 - name: Copy get-pip.py
   copy:
     src: get-pip.py
-    dest: ~/get-pip.py
```

Comment:
@asiragusa using `$HOME` env var seems to be not idiomatic for Ansible.

Did you try `{{ lookup('env', 'HOME') }}`?